onnx/defs/math/defs.cc

```diff
@@ -887,7 +910,8 @@ ONNX_OPERATOR_SET_SCHEMA(
              "tensor(uint32)",
              "tensor(uint64)",
              "tensor(int32)",
-             "tensor(int64)"},
+             "tensor(int64)",
+             "tensor(bfloat16)"},
```

Comment:
Not sure if bfloat16 should be added in the beginning of this list. @linkerzhang, any comment?